src/api/public/apidocs-new/paths/build_project.yaml

```diff
@@ -0,0 +1,99 @@
+get:
+  summary: Get a simple directory listing of all repositories for the specified project
+  description: Get a simple directory listing of all repositories for the specified project
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
+      name: project
+      schema:
+        type: string
+      required: true
+      description: Name of the project.
+      example: 'home:foo'
```

Comment:
This parameter is defined twice in this file and, for sure, it's going to be used in many other endpoints. I suggest defining it in its own file and add a reference wherever it's needed.
 
 In fact, in [the PR I've just opened](https://github.com/openSUSE/open-build-service/pull/10975/commits/cd04d7327e0ff1e1831c6d3fcac3a22a586d95ed) I did it, but I used the name `project_name` instead of `project`. Why? Because my endpoints use project, repository and architecture parameters, and there was a `architecture_name` already defined.
 
I think we should agree on using `*_name` or not but be consistent all over the documentation.

Reply:
@saraycp please have another look, applied for `project_name`, but can not reuse the repository parameter because in my case its a query parameter not a path param